parsing/parchmint-parser.js

```diff
@@ -546,6 +652,28 @@ class ParchmintParser {
         return true;
     }
 
+    /**
+     * Clear all data from the parser.
+     *
+     * Reset the parser to its beginning state. This method does not reallocate
+     * anything except the layers array.
+     *
+     * @since 1.0.0
+     */
+    clear() {
+        this.parchmint = Validation.DEFAULT_STR_VALUE;
+
+        this.valid = true;
+        this.idSet.clear();
+
+        this.architecture = null;
+        this.layers = [];
+        this.components.clear();
+        this.connections.clear();
+        this.compFeatures.clear();
+        this.connFeatures.clear();
+    }
```

Comment:
We can get rid of it if you think it would make the code cleaner. I did it this way because I thought creating and destroying objects over and over might be too intense for the browser. But now that I think about it there's probably very little difference between the two.

Reply:
Yeah I don't think the allocation is really going to cause you issues so you can remove it.

Reply:
It's been removed.